.github/workflows/Rebuild Database Templates.yml

```diff
@@ -141,11 +141,14 @@ jobs:
         $config.packages.'EdFi.Ods.CodeGen'.PackageVersion = $packageversion
         $config | ConvertTo-Json | Format-Json | Out-File -FilePath $filePath -Encoding UTF8
     - name: Commit and push updated EdFi.Ods.CodeGen package reference
-      working-directory: ./Ed-Fi-ODS-Implementation/
-      run: |
-        git add ./configuration.packages.json
-        git commit -m "Updating for new CodeGen version"
-        git push -u origin ${{ env.current_branch }}
+      uses: actions/checkout@v3
```

Comment:
I am aware of it .   I am trying to use v3 to validate that  since it's not working in  uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2